nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/MarshallerDescriptor.java

```diff
@@ -26,24 +26,23 @@
 
 import org.nuxeo.common.xmap.annotation.XNodeList;
 import org.nuxeo.common.xmap.annotation.XObject;
+import org.nuxeo.runtime.model.Descriptor;
 
 /**
  * @since 5.8
  */
 @XObject("marshaller")
-public class MarshallerDescriptor {
+public class MarshallerDescriptor implements Descriptor {
 
     @XNodeList(value = "writer", componentType = Class.class, type = ArrayList.class)
-    private List<Class<? extends MessageBodyWriter<?>>> writers;
+    public List<Class<? extends MessageBodyWriter<?>>> writers = new ArrayList<>(0);
```

Comment:
Remove the `0`, that's a micro-optimization and the default is good enough (principle of least surprise)